.travis.yml

```diff
@@ -12,23 +10,33 @@ matrix:
         - if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then
             bash -x tools/lint-pr-commit-message.sh ${TRAVIS_PULL_REQUEST};
           fi
+
+
     - name: "Linter"
       language: node_js
       node_js: "node"
       env:
         - NODE=$(which node)
+      install:
+        - make lint-py-build || true
       script:
         - make lint
+
+
     - name: "Test Suite"
+      language: cpp
+      cache: ccache
       addons:
         apt:
           sources:
             - ubuntu-toolchain-r-test
           packages:
-            - g++-6
+            - g++-8
+      env:
+        - PARALLEL_ARGS="--flaky-tests=skip --report --time"
```

Comment:
See https://github.com/nodejs/node/pull/26968 re. use of `PARALLEL_ARGS`.

Reply:
What a bowl of spaghetti 🤦‍♂️